### PR TITLE
Added link to the banner of Hacktoberfest

### PR DESCRIPTION
**Pull request by thomasbnt**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Just adding a link to https://www.hacksquad.dev on the banner which is in the README to have a better context.

### CODE REVIEW

**Comment by scopsy** [comment]
@iampearceman I actually think that you want to send it to the new language page right? If so could you provide the link to @thomasbnt so he can add the appropriate link?

**Comment by eelliot908** [comment]
You Dey jones

**Review by eelliot908 — COMMENTED** [review]
I'm not aware

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   
-![hacktoberfest-readme-dark](https://user-images.githubusercontent.com/63902456/191170980-c010daeb-f8ff-454d-bda1-543695b32c80.jpeg#gh-dark-mode-only)
+[![hacktoberfest-readme-dark](https://user-images.githubusercontent.com/63902456/191170980-c010daeb-f8ff-454d-bda1-543695b32c80.jpeg#gh-dark-mode-only)](https://www.hacksquad.dev)
 
 </div>
 
 <div align="center">
   
- ![hacktoberfest-readme-light](https://user-images.githubusercontent.com/63902456/191171065-d8f349e5-7250-43ff-8078-fa02dd859bc8.jpeg#gh-light-mode-only)
+ [![hacktoberfest-readme-light](https://user-images.githubusercontent.com/63902456/191171065-d8f349e5-7250-43ff-8078-fa02dd859bc8.jpeg#gh-light-mode-only)](https://www.hacksquad.dev)
```
Comment (thomasbnt) [inline README.md:9]: Hmm I just seen the theme wasn't applied with the markdown has a link. 
https://github.com/thomasbnt/novu/blob/added-link-to-banner-hacktoberfest-thomasbnt/README.md

**Review by p-fernandez — APPROVED** [review]
🌟 
@iampearceman if @thomasbnt has already implemented the links you provided we are good to go here

**Comment by scopsy** [comment]
@thomasbnt could you please resolve conflict from main? we updated the banner image URL with a different text. And also, could you make the link go to: https://novu.co/hacktoberfest/ instead of hacksquad?

**Comment by thomasbnt** [comment]
> @thomasbnt could you please resolve conflict from main? we updated the banner image URL with a different text. And also, could you make the link go to: [novu.co/hacktoberfest](https://novu.co/hacktoberfest/) instead of hacksquad?

Hello ! Changed to https://novu.co/hacktoberfest/ and replaced images URL 👌🏼

**Review by scopsy — APPROVED** [review]
Thank you so much for your contribution! Really appreciate it 🙏

**Pull request merged**